admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,283 @@
+			[create delete get list update watch]	[resourcegroup:exposedkube resourcegroup:exposedopenshift secrets]								[]
+			[get list watch]			[projects resourcegroup:allkube resourcegroup:allkube-status resourcegroup:allopenshift-status]					[]
+self-provisioner	Verbs					Resources															Resource Names	Extension
+			[create]				[projectrequests]														[]
+system:component	Verbs					Resources															Resource Names	Extension
+			[*]					[*]																[]
+system:delete-tokens	Verbs					Resources															Resource Names	Extension
+			[delete]				[oauthaccesstokens oauthauthorizetokens]											[]
+system:deployer		Verbs					Resources															Resource Names	Extension
+			[*]					[*]																[]
+system:registry		Verbs					Resources															Resource Names	Extension
+			[delete get]				[images]															[]
+			[get]					[imagestreamimages imagestreams imagestreamtags]										[]
+			[create update]				[imagestreams]															[]
+			[create]				[imagerepositorymappings imagestreammappings]											[]
+system:router		Verbs					Resources															Resource Names	Extension
+			[list watch]				[endpoints routes]														[]
+view			Verbs					Resources															Resource Names	Extension
+			[get list watch]			[projects resourcegroup:allkube resourcegroup:allkube-status resourcegroup:allopenshift-status resourcegroup:exposedopenshift]	[]
+----
+====
+
+.Describing the `master:default` `*policyBinding*`
+====
+
+[options="nowrap"]
+----
+$ osc describe --namespace=master policyBinding master:default
+Name:						master:default
+Created:					5 minutes ago
+Labels:						<none>
+Last Modified:					2015-05-04 18:22:57 +0000 UTC
+Policy:						master
+RoleBinding[basic-users]:
+						Role:	basic-user
+						Users:	[]
+						Groups:	[system:authenticated]
+RoleBinding[cluster-admins]:
+						Role:	cluster-admin
+						Users:	[]
+						Groups:	[system:cluster-admins]
+RoleBinding[cluster-status-binding]:
+						Role:	cluster-status
+						Users:	[]
+						Groups:	[system:authenticated system:unauthenticated]
+RoleBinding[self-provisioners]:
+						Role:	self-provisioner
+						Users:	[]
+						Groups:	[system:authenticated]
+RoleBinding[system:components]:
+						Role:	system:component
+						Users:	[system:kube-client system:openshift-client]
+						Groups:	[system:nodes]
+RoleBinding[system:delete-tokens-binding]:
+						Role:	system:delete-tokens
+						Users:	[]
+						Groups:	[system:authenticated system:unauthenticated]
+RoleBinding[system:deployers]:
+						Role:	system:deployer
+						Users:	[system:openshift-deployer]
+						Groups:	[]
+RoleBinding[system:registrys]:
+						Role:	system:registry
+						Users:	[]
+						Groups:	[system:registries]
+RoleBinding[system:routers]:
+						Role:	system:router
+						Users:	[]
+						Groups:	[system:routers]
+----
+====
+
+== Binding Users to Roles
+You can bind users to roles using `osadm policy` commands.
+
```

Comment:
I'm actually not too sure what "binding a use to a role" means. The roles are mentioned below, but there's no method for assigning a user a role. Is it just assigning a role to a user, much like in v2? Would it be easier to just state that? Say retitleing the section to "Assigning Users a Role" or something?